js/utils/motebehovUtils.js

```diff
@@ -144,15 +130,89 @@ export const finnVirksomhetnrListeMedSkalViseMotebehov = (oppfolgingsforlopsPeri
     return liste;
 };
 
-export const skalViseMotebehovMedOppfolgingsforlopListe = (oppfolgingsforlopsPerioderReducerListe, motebehovReducer) => {
+export const orgnummerFraMote = (moteReducer) => {
+    const moteData = moteReducer.data;
+    const deltakerMedOrgnummer = moteData.deltakere && moteData.deltakere.find((deltaker) => {
+        return !!deltaker.orgnummer;
+    });
+    return deltakerMedOrgnummer.orgnummer;
+};
+
+export const riktigOppfolgingsforlopsPeriodeReducer = (oppfolgingsforlopsPerioderReducerListe, orgnummer) => {
+    return oppfolgingsforlopsPerioderReducerListe.find((periodeReducer) => {
+        return periodeReducer.virksomhetsnummer === orgnummer;
+    });
+};
+
+export const moteOpprettetIOppfolgingstilfelle = (moteReducer, oppfolgingsforlopsPerioderReducerListe) => {
```

Comment:
Pirk: Personlig synes jeg det er litt lettere å lese funksjoner som returnerer true/false om de starer med et "spørre-ord" ala "is" is Java. Da vet man uten å sjekke at det ikke returneres et objekt/undefined.
Gjelder også moteplanleggerBruktIOppfolgingstilfelle .